### DISCUSSION TITLE
Move templates from Logstash to ES

### DISCUSSION THREAD

**Discussion by shipler** [Security-Onion-Solutions/securityonion#1541] [Unsupported Versions] [answered]
Parsing templates from the /elasticsearch/templates/so/*.jinja are loaded into logstash instead of elasticsearch (docker image is also not mounting the templates folder), is it on purpose due to so elasticsearch implementation?

**Comment by TOoSmOotH** [comment, marked as answer]
It's an order of operations thing. We need Elastic to be up and running before we apply the templates. We don't want to wait for elastic in the ES state as we can be doing running other states while it is loading up to speed up the overall checkin time.
